clang/lib/Sema/SemaSPIRV.cpp

```diff
@@ -16,88 +16,90 @@ namespace clang {
 
 SemaSPIRV::SemaSPIRV(Sema &S) : SemaBase(S) {}
 
+/// Checks if the first `NumArgsToCheck` arguments of a function call are of
```

Comment:
@spall  did a bunch of work to get semantic checking to be more consistent. She should review at least this file for this PR.